src/pages/lists/status-list/index.tsx

```diff
@@ -1,78 +1,371 @@
+import List from '@material-ui/core/List';
+import Hidden from '@material-ui/core/Hidden';
+import IconButton from '@material-ui/core/IconButton';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import useTheme from '@material-ui/core/styles/useTheme';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Badge from '@material-ui/core/Badge';
 import HomeIcon from '@material-ui/icons/Home';
 import MenuIcon from '@material-ui/icons/Menu';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import Chevron from '@material-ui/icons/ChevronRight';
+import NotificationsActiveIcon from '@material-ui/icons/NotificationsActive';
 import NotificationIcon from '@material-ui/icons/Notifications';
-import WarningIcon from '@material-ui/icons/Warning';
+import HelpIcon from '@material-ui/icons/Help';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
 import { useDispatch } from 'react-redux';
 import { TOGGLE_DRAWER } from '../../../redux/actions';
-import { InfoListItem, ListItemTag, InfoListItemProps } from '@pxblue/react-components';
+import { InfoListItem, ListItemTag, InfoListItemProps, Spacer } from '@pxblue/react-components';
 import * as colors from '@pxblue/colors';
+import { Maintenance } from '@pxblue/icons-mui';
+import clsx from 'clsx';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme: Theme) => ({
     appbarRoot: {
         padding: 0,
     },
     toolbarGutters: {
-        padding: '0 16px',
+        padding: `0 ${theme.spacing(2)}px`,
+        display: 'flex',
+        justifyContent: 'space-between',
+    },
+    listItemTextWithoutIcon: {
+        marginLeft: -theme.spacing(7),
+    },
+    listItemText: {
+        marginLeft: '0px',
+    },
+    accordionContainer: {
+        maxWidth: 768,
+        paddingTop: theme.spacing(3),
+        margin: '0 auto',
+
+        [theme.breakpoints.down('sm')]: {
+            margin: `0 0 ${theme.spacing(3)}px 0`,
+            paddingTop: 0,
+        },
+    },
+    accordionRoot: {
+        marginBottom: theme.spacing(2),
+        borderRadius: '4px',
+        '&:before': {
+            display: 'none',
+        },
+        '& .MuiAccordionSummary-root': {
+            height: '48px',
+            minHeight: '48px',
+            '&.Mui-expanded': {
+                borderBottom: `1px solid ${theme.palette.divider}`,
+            },
+        },
+        [theme.breakpoints.down('sm')]: {
+            maxWidth: '100%',
+            boxShadow: 'none',
+            borderRadius: 0,
+            '&:before': {
+                display: 'none',
+            },
+            '& .MuiAccordionSummary-root': {
+                height: '48px',
```

Comment:
Always avoid absolute pixel values like this for spacing

Reply:
Done.